pandas/tests/io/parser/conftest.py

```diff
@@ -63,21 +68,27 @@ def csv1(datapath):
 _cParserHighMemory = CParserHighMemory()
 _cParserLowMemory = CParserLowMemory()
 _pythonParser = PythonParser()
+_pyarrowParser = PyArrowParser()
 
 _py_parsers_only = [_pythonParser]
 _c_parsers_only = [_cParserHighMemory, _cParserLowMemory]
-_all_parsers = [*_c_parsers_only, *_py_parsers_only]
+_pyarrow_parsers_only = [_pyarrowParser]
 
 _py_parser_ids = ["python"]
 _c_parser_ids = ["c_high", "c_low"]
-_all_parser_ids = [*_c_parser_ids, *_py_parser_ids]
+_pyarrow_parser_ids = ["pyarrow"]
+
+_all_parsers = [*_c_parsers_only, *_py_parsers_only, *_pyarrow_parsers_only]
+_all_parser_ids = [*_c_parser_ids, *_py_parser_ids, *_pyarrow_parser_ids]
 
 
 @pytest.fixture(params=_all_parsers, ids=_all_parser_ids)
 def all_parsers(request):
     """
     Fixture all of the CSV parsers.
     """
+    if request.param.engine == "pyarrow":
+        pytest.importorskip("pyarrow", "0.15.0")
```

Comment:
is this min version mentioned anywhere in doc-strings?

Reply:
No and this should just be removed

When this PR started pyarrow required version for this was higher than the repo-wide minimum. We have since bumped the minimum pyarrow version so no need for special handling here

Reply:
@jreback @arw2019 
The min version for pandas in general is 0.15.0, and this code was intended to skip pyarrow tests for machines that don't have the required version of pyarrow installed.

Reply:
to be honest for 1.3 we should just bump to 1.0 anyhow to avoid all of these complexites. let me make an issue.

Reply:
> @jreback @arw2019
> The min version for pandas in general is 0.15.0, and this code was intended to skip pyarrow tests for machines that don't have the required version of pyarrow installed.

Right in which case we want to use the global min version here instead of hard coding for whenever we decide to bump the global min

Reply:
Yeah, no disagreements there, we can probably pull this from the dependencies version dict. However, without this, pyarrow tests will fail on machines where it isn't installed I think.